sdks/typescript/src/step.ts

```diff
@@ -245,15 +257,56 @@ export class Context<T, K = {}> {
     await this.client.event.putStream(stepRunId, data);
   }
 
+  /**
+   * Enqueues multiple children workflows in parallel.
+   * @param children an array of objects containing the workflow name, input data, and options for each workflow
+   * @returns a list of workflow run references to the enqueued runs
+   */
+  bulkEnqueueChildren<Q extends Record<string, any> = any, P extends Record<string, any> = any>(
+    children: Array<{
+      workflow: string | Workflow | WorkflowV1<Q, P>;
+      input: Q;
+      options?: {
+        key?: string;
+        sticky?: boolean;
+        additionalMetadata?: Record<string, string>;
+      };
+    }>
+  ): Promise<WorkflowRunRef<P>[]> {
+    return this.spawnWorkflows(children);
+  }
+
+  /**
+   * Runs multiple children workflows in parallel.
+   * @param children an array of objects containing the workflow name, input data, and options for each workflow
+   * @returns a list of results from the children workflows
+   */
+  async bulkRunChildren<Q extends Record<string, any> = any, P extends Record<string, any> = any>(
+    children: Array<{
+      workflow: string | Workflow | WorkflowV1<Q, P>;
+      input: Q;
+      options?: {
+        key?: string;
+        sticky?: boolean;
+        additionalMetadata?: Record<string, string>;
+      };
+    }>
+  ): Promise<P[]> {
+    const runs = await this.bulkEnqueueChildren(children);
+    const res = runs.map((run) => run.result());
+    return Promise.all(res);
+  }
+
   /**
    * Spawns multiple workflows.
    *
    * @param workflows an array of objects containing the workflow name, input data, and options for each workflow
    * @returns a list of references to the spawned workflow runs
+   * @deprecated use bulkEnqueueChildren or bulkRunChildren instead
    */
-  spawnWorkflows<Q = JsonValue, P = JsonValue>(
```

Comment:
nit: on the Python side I've been calling this `Record<string, any>` `JSONSerializableMapping` to make it clear what we're expecting - I kinda like that type of naming (i.e. `JsonValue`)